libs/domain/user/summary/src/summary.component.ts

```diff
@@ -26,14 +29,64 @@ export class UserSummaryComponent extends ContentMixin(LitElement) {
     resolve(SemanticLinkService).get({ type: SemanticLinkType.Login })
   );
 
-  protected override render(): TemplateResult | void {
-    return html` <oryx-button>
-      <a href=${ifDefined(this.link)}>
-        <oryx-icon type="user"></oryx-icon>
-        <oryx-heading tag=${HeadingTag.Subtitle} .maxLines=${1}>
-          ${this.user?.firstName} ${when(!this.user, () => i18n('user.login'))}
-        </oryx-heading>
-      </a>
-    </oryx-button>`;
+  protected renderTriggerButton(isAuthenticated: boolean): TemplateResult {
+    const innerContent = html`<oryx-icon type="user"></oryx-icon>
+      <oryx-heading tag=${HeadingTag.Subtitle} .maxLines=${1}>
+        ${when(
+          isAuthenticated,
+          () => this.user?.firstName,
+          () => i18n('user.login')
+        )}
+      </oryx-heading>`;
+
+    return html`
+      <oryx-button>
+        ${when(
+          isAuthenticated,
+          () => html`<button class="trigger">${innerContent}</button>`,
+          () =>
+            html`<a href=${ifDefined(this.link)} class="trigger">
+              ${innerContent}
+            </a>`
+        )}
+      </oryx-button>
+    `;
+  }
+
+  protected renderDropdown(): TemplateResult {
+    return html`
+      <div class="dropdown">
+        ${when(this.options?.items && this.options.items.length, () =>
+          this.options!.items!.map(
+            (item) => html`
+              <oryx-button type="text">
+                <a href=${item.link} class="dropdown-link" close-popover>
+                  ${when(
+                    item.icon,
+                    () => html`<oryx-icon type="${item.icon}"></oryx-icon>`
+                  )}
+                  ${item.title}
+                </a>
+              </oryx-button>
+            `
+          )
+        )}
+        <auth-logout></auth-logout>
+      </div>
+    `;
+  }
+
+  protected override render(): TemplateResult {
+    if (!this.user) {
+      return this.renderTriggerButton(false);
+    }
+
+    return html`
+      <oryx-dropdown position="start" vertical-align>
+        <div slot="trigger">${this.renderTriggerButton(true)}</div>
```

Comment:
Do we really need this additional wrapper for trigger? Can we set `slot="trigger"` on oryx-button directly?